runtime_k3d/test/integration/integration_test.py

```diff
@@ -91,4 +91,3 @@ def test_scripts_run_successfully():
     assert check_image_if_created(image_reg)
     assert run_command(f"{BASE_COMMAND_DEPLOYMENT} deploy-vehicleapp")
     assert check_pods(pods_regs)
-    assert run_command(f"{BASE_COMMAND_RUNTIME} down")
```

Comment:
We should still test if the runtime can be pulled down successfully. Please don't just remove tests, adapt them accordingly